CHANGES_NEXT_RELEASE

```diff
@@ -23,3 +23,4 @@
 - Fix: invalid regex patterns detection (#968)
 - Fix: Better error returned on invalid geoquery (Issue #2174)
 - Fix: Better error returned on attribute not found in GET /v2/entities/<entity-id>/attrs/<attr-name>/value (Issue #2220)
+- Fix: Correct error returned when trying to replace a non-existing attribute (Issue #2221)
```

Comment:
Maybe the operation that has been fixed (PUT /v2/entities/id/attrs/attrName) should be mentioned?